package.json

```diff
@@ -11,12 +11,12 @@
   "main": "server.js",
   "dependencies": {
     "async": "*",
-    "americano": "=0.3.3",
-    "americano-cozy": "=0.2.2",
-    "cozy-realtime-adapter": "=0.12.0"
+    "americano": "=0.4.2",
```

Comment:
@clochix IIRC, you have to also update server.coffee if you upgrade americano. The callback for start takes one more argument - err - that comes first. See https://github.com/cozy/americano#handmade for an example